Sources/AardvarkReveal/ARKCompressionUtility.h

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
I'm tempted to put this utility into the `Aardvark` module, since it isn't specific to the Reveal attachment generation. I could see it being useful for other attachments as well (even being able to compress the log stores, to reduce the data size of bug reports).